features/polyExplorer/src/screens/entityDetails/companyRevenueChart/companyRevenueBarChartInfo.jsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
+
+import i18n from "../../../i18n.js";
+import StoriesInfoScreen from "../../../components/clusterStories/storiesInfoScreen.jsx";
+import Infographic from "../../../components/infographic/infographic.jsx";
+
+const CompanyRevenueBarChartInfo = () => {
+    const CompanyRevenueBarChartInfoContent = [
+        <div className="base-info-padding">
+            {/* <p>{i18n.t("companiesBarChartInfoScreen:p1")}</p> */}
```

Comment:
Still waiting for some texts here from Sven (tomorrow), that's why I kept it outcomented